kubetest/anywhere_test.go

```diff
@@ -30,20 +30,29 @@ func TestNewKubernetesAnywhere(t *testing.T) {
 
 	cases := []struct {
 		phase2            string
+		cniName           string
 		expectConfigLines []string
 	}{
 		{
-			phase2: "kubeadm",
+			phase2:  "kubeadm",
+			cniName: "weave",
 			expectConfigLines: []string{
 				".phase2.provider=\"kubeadm\"",
-				".phase3.weave_net=y",
+				".phase3.cni_name=\"weave\"",
+			},
+		},
+		{
+			phase2:  "kubeadm",
```

Comment:
Now that the phase2 string isn't unique per test case, can you update all of the `t.Error()` calls to make it clear which specific case failed?

The existing lines look like:

    t.Errorf("NewKubernetesAnywhere(%s) some failure...", tc.phase2, ...)

but it should either report both phase2 + cni or a new `name` field to describe the test, like we do [elsewhere](https://github.com/kubernetes/test-infra/blob/b4e81460cef97a836a93dd8f0947ebbc1f2b0239/kubetest/util_test.go#L45).